src/SeaIceDynamics/stress_balance_free_drift.jl

```diff
@@ -0,0 +1,88 @@
+abstract type AbstractFreeDriftDynamics end
+
+"""
+    StressBalanceFreeDrift{T, B, C}
+
+A free drift parameterization that computes the free drift velocities as a balance between
+top and bottom stresses ``τa ≈ τo`` where we split the stresses into a linear implicit part
+and an explicit part  ``τaˣ = τaˣₑ + u * τaˣᵢ`` and ``τoˣ = τoˣₑ + u * τoˣᵢ`` (and similarly for the y-component) such that 
```

Comment:
the units don't make sense.

Reply:
`τ` has units m^2 / s^2, `u` is m/s, and so if you want something of the form `τ = something * u`, then `something` has units m/s --- its a velocity.

Is that what you want? Not a quadratic term?

Reply:
right, I will change the docs and the names in the code to make sure it is more understandable.
I am not sure this is necessarily what we want, this is how we treat the stress in the `SeaIceMomentumEquations` (semi-implicitly), but maybe here we can treat it quadratically. I ll need to add a couple more methods.

Reply:
It might be simpler if one of the two stresses does not depend on the ice velocity, so that we can write an explicit formula. I ll add a method for when the top stress is independent on the ice velocity and when the bottom stress is independent of the ice velocity. We can leave the case in which both stresses are implicit for which we need a non-linear solve for the future.